project/Dependencies.scala

```diff
@@ -52,7 +52,6 @@ object Dependencies {
   val jacksonParameterNames = "com.fasterxml.jackson.module" % "jackson-module-parameter-names" % JacksonVersion
   val jacksonScala = "com.fasterxml.jackson.module" %% "jackson-module-scala" % JacksonVersion
 
-  val testcontainers = "org.testcontainers" % "testcontainers" % TestContainersVersion
```

Comment:
I liked `testcontainers` over `testContainers` as it's written without a - in the artifact name, but :+1:

Reply:
Yes, the official name is `Testcontainers` as one word. I removed the dependency that wasn't being used. Didn't think of switching to this one.